README.md

```diff
@@ -11,29 +11,31 @@ Status](https://api.netlify.com/api/v1/badges/b8310579-ac4f-41f2-a144-9c90fca9b3
 
 ## Introduction
 
-[Volto](https://github.com/plone/volto) is a React-based frontend for content
-management systems, currently supporting three backend implementations: Plone,
-Guillotina and a NodeJS reference implementation.
+[Volto](https://github.com/plone/volto) is a [React](https://reactjs.org/)-based [content management system (CMS)](https://en.wikipedia.org/wiki/Content_management_system).
 
-[Plone](https://plone.org) is a CMS built on Python with over 17 years of experience.
+### Plone Backend
 
-Plone has very interesting features that appeal to developers and users alike,
-such as customizable content types, hierarchical URL object traversing and a
-sophisticated content workflow powered by a granular permissions model. This
-allows you to build anything from simple websites to enterprise-grade
-intranets.
+Volto uses [Plone](https://plone.org) and its [REST API](https://github.com/plone/plone.restapi) as a backend.
+Plone is a mature and secure content management system written in [Python](https://www.python.org/).
+It has the [best security track record of any major CMS](https://plone.com/secure.html).
+Out-of-the-box, Plone comes with everything you need to build sophisticated websites and enterprise-grade intranets.
```

Comment:
Sorry, but we are doing it again. We are overselling Plone over the subject of the writing. We are speaking now about Volto, right? So Plone can't have more sentences describing it than Volto has. Volto has plenty of features worth to be mention above the fact that the backend is Plone.

Reply:
+1

Reply:
@sneridagh you are absolutely correct. I think I have to rewrite the entire thing from the scratch again to get it right. It is so hard to get Plone out of our heads here...